src/RAI.jl

```diff
@@ -95,6 +95,8 @@ export
 include("creds.jl")
 include("config.jl")
 include("rest.jl")
+include("gen/relationalai/protocol/Protocol_PB.jl")
```

Comment:
I guess shouldn't this be including `Relationalai_PB.jl` instead?
I guess maybe you could do this:
```suggestion
include("gen/relationalai/Relationalai_PB.jl")
using .Relationalai_PB: Protocol_PB
```